src/test/scala/fr/acinq/bitcoin/BIP49Spec.scala

```diff
@@ -11,7 +11,7 @@ import scodec.bits._
   */
 class BIP49Spec extends FunSuite {
   test("BIP49 reference tests") {
-    val seed = MnemonicCode.toSeed("abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about".split(" "), "")
+    val seed = MnemonicCode.toSeed("abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about".split(" ").toSeq, "")
```

Comment:
```suggestion
    val seed = MnemonicCode.toSeed("abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about", "")
```

Reply:
Fixed in https://github.com/ACINQ/bitcoin-lib/pull/44/commits/7d3622c628c7b1ff51ccf69d7aaae6ce4d664e67